radio/src/opentx.cpp

```diff
@@ -1020,8 +1020,12 @@ void checkTrims()
 }
 
 #if !defined(SIMU)
+#if defined(FLYSKY_HALL_STICKS)
+uint32_t s_anaFilt[NUM_ANALOGS];
```

Comment:
Shouldn’t 16 bits be enough to store a 11/12bits value?

Reply:
The filtering sums 16 times a value. For a 12-bit value, this is perfectly fine.
For FlySky, that filtering output goes over 65535 (tested on radio, will clip), as the range is more than 4500 minimally, thus 16 times 4500 will be more than 16-bit.

Reply:
Ok, then just change the type for everyone, no need to stay on 16bits then. No need for `#if defined...` then ;-)

Reply:
OK, will do.